docs/UserGuide.md

```diff
@@ -260,13 +262,34 @@ Example:
 
 ![UiFind](images/UG-screenshots/UiFind.png)
 
-Replace the prefix n/ with the corresponding prefixes (p/, np/, e/, a/, dt/ or t/) for find by other fields.
+Replace the prefix `n/` with the corresponding prefixes (`p/`, `np/`, `e/`, `a/`, `dt/` or `t/`) for find by other fields.
 The same rules apply as per find by name.
 
 [Back to top](#table-of-contents)
 
 ---
 
+### Sort the displayed students: 'sort'
+
+Sort the displayed list of students (in the Student's Details panel) by the given `type` and `order`.
+
+Format: `sort TYPE [ORDER]`
+
+- `TYPE` must be from `NAME`, `CLASS`, and `OWED`.
+- `ORDER`, if present, must be from `ASC` or `DESC`. If it is left blank, it will follow the default order for the type.
+- Default order for `NAME` and `CLASS` is `ASC` while default order for `OWED` is `DESC`.
+- The two parameters `TYPE` and `ORDER` can be non-capitalized such as `name`, `nAMe` and so on.
+
+Examples:
+
+- `list` followed by `sort NAME` will list all the students by ascending order of their names.
```

Comment:
Think can word as:
`list` followed by `sort NAME` will list all the students according to their names in ascending order.
similar for below